### DISCUSSION TITLE
Editor Window keeps making a new copy of itself.

### DISCUSSION THREAD

**Discussion by Led54** [apohl79/audiogridder#1187] [Help]
Hi, Thank you so much for this awesome utility.
Every time I load a plugin the editor window keeps making a new copy of itself. 
Otherwise it works perfectly. 
Using the generic editor window is not an option for me because I need to be able to edit the parameters of the plugin.
Any help would be greatly appreciated

![Error 2](https://user-images.githubusercontent.com/127719708/224624828-609dce6f-ae39-4e49-a96d-c7aabf9472af.JPG)
![Error Report](https://user-images.githubusercontent.com/127719708/224624840-baae08ac-4a8d-4c07-a809-49811344f43e.JPG)

**Comment by apohl79** [comment]
If you would be able to follow those steps: https://audiogridder.com/bug-reports/

I might be able to see whats going on.
